src/gis/components/Map.js

```diff
@@ -254,11 +254,6 @@ const LeafletSearch = () => {
       inputField.setAttribute('aria-label', t('gis.map_search_placeholder'));
       inputField.setAttribute('role', 'combobox');
 
-      const clearButton =
```

Comment:
Question: Why is this not needed anymore?

Reply:
This was an accessibility patch I added. The leaflet search people fixed the module so it now uses a button instead of a link. As a result my patch (a) is no longer needed (b) causes an error.